roseau/load_flow/models/lines/parameters.py

```diff
@@ -398,13 +555,13 @@ def from_geometry(
         id: Id,
         *,
         line_type: LineType,
-        conductor_type: ConductorType | None = None,
-        insulator_type: InsulatorType | None = None,
+        material: Material | None = None,
+        insulator: Insulator | None = None,
```

Comment:
Use the plural names to avoid a future breakage when/if we have lines with different material/insulator in the catalogue.

Alternativaly, we can have `material`/`material_neutral` and `insulator`/`insulator_neutral` similar to `section`/`section_neutral`. If you choose this option, we can add these parameters later; no change is needed now.
```suggestion
        materials: Material | None = None,
        insulators: Insulator | None = None,
```

Reply:
I have added all the `neutral` elements in the catalogue (data, `get_catalogue` and `from_catalogue`) and in the `from_geometry` class method.